recipe/meta.yaml

```
@@ -1,29 +1,28 @@
 {% set name = "pyrfr" %}
-{% set version = "0.9.0" %}
-
+{% set version = "0.8.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: bc6e758317cf79579fe6b7ce5f01dd42f77c991bf707e33646e8c6a9112c186b
+  sha256: cdc92b6054a351b85cc7496a2fc86e3192a9d04a92fd6bfa4aa629af360a6d1d
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
```

skip needed for python bound
```suggestion
        skip: True  # [py<37]
```
https://github.com/automl/random_forest_run/blob/c066b25e8bdd23f842c3bf11dea1eefb75307e70/pyrfr/setup.py#L62